pkg/host-disk/host-disk_test.go

```diff
@@ -200,7 +200,7 @@ var _ = Describe("HostDisk", func() {
 					Expect(true).To(Equal(os.IsNotExist(err)))
 				})
 
-				It("Should NOT subtract reserve if there is enough space on storage for requested size", func(done Done) {
+				It("Should NOT subtract reserve if there is enough space on storage for requested size", func() {
```

Comment:
This is not following migration guide https://onsi.github.io/ginkgo/MIGRATING_TO_V2#migration-strategy, are we ok with this ? 

This shoul be migrated to
```golang
It("...", func() {
	done := make(chan interface{})
	go func() {
		// user test code to run asynchronously
		close(done) //signifies the code is done
	}()
	Eventually(done, timeout).Should(BeClosed())
})
```

Reply:
All the tests are passing, they are running sync
I think it is just a recommendation for the equivalent behavior of the deprecated feature,
the feature was removed because it adds confusion (according the guide),

I think that it will be better to have one level of parallel that we already have,
ad hoc in case something is needed we can improve, but not globally just because we want to preserve the async behavior which is now officially deprecated